test/parser/parser-errors.t

```diff
@@ -125,14 +125,57 @@ mtt: Parse error: Boxed expression is expected
   mtt: Parse error: Missing or unexpected lexeme in parenthesized expression
   [1]
 
-$ mtt parse -e "let x = () x"
-mtt: Parse error: Missing or unexpected lexeme in parenthesized expression
-[1]
+  $ mtt parse -e "let x = () x"
+  mtt: Parse error: Missing or unexpected lexeme in parenthesized expression
+  [1]
 
-$ mtt parse -e "let x = () y = x"
-mtt: Parse error: Missing or unexpected lexeme in parenthesized expression
-[1]
+  $ mtt parse -e "let x = () y = x"
+  mtt: Parse error: ZERO token can be used in match-expression only
```

Comment:
This error message is misleading

Reply:
Yeah. I discovered that this error is very strange, because it comes from rule `UNIT PLUS ZERO`. Even if we replace `()` with something else, we will get the same error